INSTALL.md

```diff
@@ -1413,7 +1413,7 @@ be tested.  Run:
 **Warning:** you MUST run the tests from an unprivileged account (or disable
 your privileges temporarily if your platform allows it).
 
-See [test/README.md](test/README.md) for further details how run tests.
+See [test/README.md](test/README.md) for further details on how to run tests.
```

Comment:
Agreed `on` and `to` are missing, but so is `the` before `tests` (compare line 1413)